src/monodroid/jni/monodroid-glue.c

```diff
@@ -1241,6 +1249,12 @@ ensure_jnienv (void)
 	return env;
 }
 
+JNIEnv*
+get_jnienv (void)
```

Comment:
`ensure_jnienv()` is static and thus will be inlined in the local calls generating faster code, `get_jnienv()` won't be inlined and is meant to be used by code outside `monodroid-glue.c`. It doesn't add any significant overhead to the code and it still saves a function call for the local `monodroid-glue.c` code.